src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int height;
+    private int firstLeg;
+    private int secondLeg;
+
+    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, Color color) {
+        super(color);
+        this.setFirstLeg();
+        this.setSecondLeg();
+        this.setHeight();
+    }
```

Comment:
```suggestion
    public IsoscelesTrapezoid(int firstLeg, int secondLeg, int height, Color color) {
        super(color);
        this.height = height;
        this.secondLeg = secondLeg;
        this.firstLeg = firstLeg;
    }
```

same for other figures